scripts/serverStat

```diff
@@ -137,6 +137,11 @@ if  [[ $NAME == *rec* ]]; then
     fi
 fi
 
+if [[ "${NAME,,}" == *drp* ]] && ([[ $CMD == "off" ]] || [[ $CMD == "cycle" ]]); then
+    printf "serverStat $CMD is not supported on the DRP nodes as the underlying psipmi power off or psipmi power cycle commands will disable the ipmi card. Instead, use the web interface as described on:\n\nhttps://confluence.slac.stanford.edu/display/PSDMInternal/Debugging+DAQ#DebuggingDAQ-IPMI\n"
```

Comment:
I'm going to repost Silke's last comment as a suggestion because her messages seem to be getting buried- maybe as a result of how Github processes email replies?

> I apparently can't English....can you change the beginning to:
serverStat $CMD is not supported for DRP nodes

```suggestion
    printf "serverStat $CMD is not supported for DRP nodes as the underlying psipmi power off or psipmi power cycle commands will disable the ipmi card. Instead, use the web interface as described on:\n\nhttps://confluence.slac.stanford.edu/display/PSDMInternal/Debugging+DAQ#DebuggingDAQ-IPMI\n"
```


https://github.com/pcdshub/engineering_tools/commit/1333e8d6c86553eddbbc37fd342e1df15dffda52#comments

I think with her last request in I will re-approve and we can merge

Reply:
I completely missed this. How can I see this comment without your hyperlink?